src/test/java/eu/elixir/ega/ebi/constants/Constants.java

```diff
@@ -0,0 +1,9 @@
+package eu.elixir.ega.ebi.constants;
+
+public interface Constants {
+
+    String KEY_FILEKEYS = "/keys/filekeys/";

```

Comment:
I think you could put this into the KeyServiceITest  class because you probably won't have to explicitly call that service from any other class.

Reply:
Done